index.js

```diff
@@ -0,0 +1,43 @@
+const ul = document.getElementById('ul-list');
+const form = document.getElementById('form');
+const inputAuthor = document.getElementById('author');
+const inputBook = document.getElementById('book');
+
+const bookDetails = JSON.parse(localStorage.getItem('bookDetails')) || [];
+
+function addBook() {
+  ul.innerHTML = '';
+  for (let i = bookDetails.length - 1; i >= 0; i -= 1) {
+    const bookInfo = bookDetails[i];
+    const newBook = document.createElement('li');
+    newBook.innerHTML = `
+      <p class="book">${bookInfo.book}</p>
+      <p class="author">${bookInfo.author}</p>
+      <button type="button" onclick="removeItem(${i})">Remove</button>
+      <hr>
+    `;
+    ul.insertBefore(newBook, ul.firstChild);
+  }
+}
+
+document.addEventListener('DOMContentLoaded', addBook);
+
+function removeItem(index) {
+  bookDetails.splice(index, 1);
+  addBook();
+  localStorage.setItem('bookDetails', JSON.stringify(bookDetails));
+}
+removeItem();
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (inputAuthor.value === '' || inputBook.value === '') {
+    return;
+  }
+  const newBooksAdded = { author: inputAuthor.value, book: inputBook.value };
+  bookDetails.unshift(newBooksAdded);
+  localStorage.setItem('bookDetails', JSON.stringify(bookDetails));
+  addBook();
+  inputAuthor.value = '';
+  inputBook.value = '';
+});
```

Comment:
As mentioned by the previous reviewer, please note that on each refresh, one of the books added to the page is removed from the page and localStorage 🤷🏽‍♂️. See the demo below:
[Screencast from 21-03-2023 23:15:14.webm](https://user-images.githubusercontent.com/44624138/226755171-c40b41ce-8608-4700-be68-54458b6dbc6c.webm)

Reply:
Hi @rotimiazeez,

Do you have maybe a hint how to solve that?. Because the removeItem() is declared and used in the innerHTML, so because that we have a linter error 'removeItem' is defined but never used  no-unused-vars. That's why we invoked the function.

Thanks for your assistance!!!!!

Reply:
Hi @J-C-S-V,
I left you a message on slack 👍🏽